crossdock/main.go

```diff
@@ -98,7 +98,9 @@ func is2xxStatusCode(statusCode int) bool {
 func httpHealthCheck(logger *zap.Logger, service, healthURL string) {
 	for i := 0; i < 240; i++ {
 		res, err := http.Get(healthURL)
-		res.Body.Close()
+		if err == nil {
```

Comment:
do we need this check?

Reply:
yes - if there is error there is no guarantee that `res` is not null.